server/server.go

```diff
@@ -132,6 +142,7 @@ func New(conf Config) (*Server, error) {
 		slacker:             conf.Slacker,
 		stillRunningMsgFreq: conf.StillRunningMsgFreq,
 		uploadTracker:       newUploadTracker(conf.Slacker, conf.SlackMessageDebounce),
+		iRodsTracker:        SlackConfig{slacker: conf.Slacker, debounceTimeout: conf.SlackMessageDebounce},
```

Comment:
I'd expect an iRodsTracker to be implemented exactly like newUploadTracker, ie. it's a new struct that takes a config in its new method. I'd possibly have expected both upload message tracking and connection message tracking to both use the same generic message tracker, if their desired functionality is similar enough.